### ISSUE TITLE
Check Metadata Links Use new Data Exchange URL

### ISSUE THREAD

**Issue by philipbaileynar** [Riverscapes/riverscapes-tools#874]
Double check that models that include URLs to the data exchange are using the new URL and not the old "warehouse" URL.

<img width="1307" alt="Screenshot 2024-01-25 at 9 07 56 PM" src="https://github.com/Riverscapes/riverscapes-tools/assets/8645379/315c81ec-8167-4eb4-af0d-39202ea64f5c">

**Comment by r-k-g** [comment]
![image](https://github.com/user-attachments/assets/0cccbfce-b003-441d-a2f0-0752878efbe1)
Newer versions of VBET seem to use the proper links. Haven't fully checked this from the code side yet.

**Comment by philipbaileynar** [comment]
I just performed a review by searching for `https://warehouse` and found nothing. I also searched the entire repo for the term `warehouse. There are lots of occurrences, but none seem to be related to building URLs. I left everything unchanged. I don't want to risk breaking things for the sake of changing a few variable names that don't matter outside of the code file in which they occur. Closing this issue.
